torchvision/models/detection/rpn.py

```diff
@@ -157,7 +156,8 @@ def forward(self, image_list, feature_maps):
         # type: (ImageList, List[Tensor])
         grid_sizes = list([feature_map.shape[-2:] for feature_map in feature_maps])
         image_size = image_list.tensors.shape[-2:]
-        strides = [[int(image_size[0] / g[0]), int(image_size[1] / g[1])] for g in grid_sizes]
+        strides = [[torch.tensor(image_size[0] / g[0], dtype=torch.int64),
+                    torch.tensor(image_size[1] / g[1], dtype=torch.int64)] for g in grid_sizes]
```

Comment:
With this change, I wonder if lines 124-126 are still necessary?
```python
            if torchvision._is_tracing():
                # required in ONNX export for mult operation with float32
                stride_width = torch.tensor(stride_width, dtype=torch.float32)
                stride_height = torch.tensor(stride_height, dtype=torch.float32)
```

Also, do we need to pass the `device` in the tensor constructor, so that it works for CUDA as well?

Reply:
Thanks a lot for pointing this out. The lines here are not needed anymore. I'll remove this part.

Reply:
can you also pass the `device` to the tensor constructor?